src/main/resources/db/migration/V2__Initial_schema.sql

```diff
@@ -0,0 +1,25 @@
+DROP TABLE IF EXISTS `user_session`;
```

Comment:
You can't do that, you would lose data on production. Updating tables are done using `ALTER TABLE` semantics https://www.w3schools.com/sql/sql_alter.asp.

Reply:
ok, thanks! implemented the suggested modification